pkgs/tools/security/nsjail/default.nix

```diff
@@ -18,6 +18,8 @@ stdenv.mkDerivation rec {
   buildInputs = [ libnl protobuf protobufc ];
   enableParallelBuilding = true;
 
+  env.NIX_CFLAGS_COMPILE = toString [ "-Wno-error" ];
```

Comment:
@happysalada for my educational purpose, why not just `"-Wno-error"` ?

Reply:
you mean why use toString and a list ? 
The idea is that it's more extensible (in the sense that if more flags need to be added later, it's just a copy and paste), but you're right, I could have just use a single string for this.
thinking about it, I could have added a comment there saying that it was required with gcc13.

Reply:
all good, than you for explanation